File: Application/Services/Operations/BenchBudgetService/Dtos/CollectDeliverCostsDto.cs
using Application.Services.Operations.BenchBudgetService.Dtos.Enums;

namespace Application.Services.Operations.BenchBudgetService.Dtos
{
    public class CollectDeliverCostsDto
    {
        public int Id { get; set; }
        public bool IsHaveCost { get; set; }
        public bool RoundTrip { get; set; } = false;
        public CostFromEnumDto CostFrom { get; set; }
        public decimal Price { get; set; }
        public decimal ApartPrice { get; set; }
    }
}

File: Views/src/components/budget-service-bench/dto/collect-deliver-costs-dto.ts
import { CostFromEnumDto } from "./enums/cost-from-enum-dto";

    export class CollectDeliverCostsDto
    {
         Id:number;
         IsHaveCost:boolean;
         RoundTrip:boolean;
         CostFrom:CostFromEnumDto;
         Price:number;
         ApartPrice:number;
    }

